packages/prover/src/cli/cmds/start/index.ts

```diff
@@ -4,7 +4,7 @@ import {proverProxyStartHandler} from "./handler.js";
 import {StartArgs, startOptions} from "./options.js";
 
 export const proverProxyStartCommand: CliCommand<StartArgs, GlobalArgs> = {
-  command: "start",
+  command: "*",
```

Comment:
I believe there is a way in `Yargs` to make any command default. So if any command is not provided that is executed, but you can still call the command with that name. 

I also agree to earlier suggestion from @nflaig to rename `start` to `proxy`.